spec/ddtrace/contrib/presto/client_spec.rb

```diff
@@ -34,6 +36,28 @@
 
   let(:presto_client_gem_version) { Gem.loaded_specs['presto-client'].version }
 
+  before(:each) do
+    timeout_seconds = 10
+    wait(timeout_seconds).for { is_presto_online }.to be true
```

Comment:
You could use `try_wait_until` here instead; there should be some other examples of its use in the codebase.